flake.nix

```diff
@@ -313,16 +319,31 @@
         }
       ];
 
+      extraHackagePackages = with nixpkgs.lib;
```

Comment:
Seems like this should be in the helper repo?

Reply:
we should rather update extraSources to this format (just a list of paths)